include/mull/JunkDetection/CXX/ASTStorage.h

```diff
@@ -37,11 +44,17 @@ class ASTStorage {
 
   ThreadSafeASTUnit *findAST(const MutationPoint *point);
 
+  MutationPointSourceInfo getSourceInfo(MutationPoint *mutationPoint);
+
+  void setSourceRange(MutationPoint *mutationPoint,
+                      clang::SourceRange sourceRange);
```

Comment:
I think it's better and probably more flexible to store an AST node here instead of the range.
Do you foresee any issues with this approach?